tests/linalg_test.py

```diff
@@ -1619,6 +1619,52 @@ def testToeplitzConstructionWithKnownCases(self):
       [2, 1, 4, 5],
       [3, 2, 1, 4]], dtype=np.float32))
 
+  @jtu.sample_product(
+    size=[1, 2, 3, 10, 100],
+    dtype=float_types + complex_types)
+  def testNullSpace(self, size, dtype):
+    rng = jtu.rand_default(self.rng())
```

Comment:
For tests, we generally treat the `jsp_linalg` function as the ground truth and check that JAX's output matches. That can make the tests simpler and more comprehensive. What do you think?

Reply:
Do you mean `scipy.linalg`? Otherwise I am not sure I understand.
But if so, I can definitely change the test to make sure it tests against a ground truth

Reply:
Yes, sorry I mean treat scipy as the ground truth

Reply:
done